esp-hal/MIGRATING-0.22.md

```diff
@@ -459,6 +471,12 @@ The Address and Command enums have similarly had their variants changed from e.g
 + Command::_1Bit
 ```
 
+`write_byte` and `read_byte` were removed and `write_bytes` and `read_bytes` can be used as replacement.
```

Comment:
You could expand on this a little bit - like mentioning to use a one byte sized buffer and checking the returned length